File: branches/BibleWebService_trunk_2.0.0.0_CSharpConversion/Book.vb
Public Class Book
    Private _BookNo As Integer
    Private _Section As String
    Private _BookText As String
    Private _Chapters As Chapters

    Public Property BookNo() As Integer
        Get
            Return _BookNo
        End Get
        Set(ByVal Value As Integer)
            _BookNo = Value
        End Set
    End Property

    Public Property Section() As String
        Get
            Return _Section
        End Get
        Set(ByVal Value As String)
            _Section = Value
        End Set
    End Property

    Public Property BookText() As String
        Get
            Return _BookText
        End Get
        Set(ByVal Value As String)
            _BookText = Value
        End Set
    End Property

	Public Property Chapters() As Chapters
		Get
			Return _Chapters
		End Get
		Set(ByVal Value As Chapters)
			_Chapters = Value
		End Set
	End Property
End Class

File: branches/BibleWebService_trunk_2.0.0.0_CSharpConversion/Book.cs
namespace BibleWebService
{
    public class Book
    {
        private int _BookNo;
        private string _BookText;
        private Chapters _Chapters;
        private string _Section;

        public int BookNo
        {
            get { return _BookNo; }
            set { _BookNo = value; }
        }

        public string Section
        {
            get { return _Section; }
            set { _Section = value; }
        }

        public string BookText
        {
            get { return _BookText; }
            set { _BookText = value; }
        }

        public Chapters Chapters
        {
            get { return _Chapters; }
            set { _Chapters = value; }
        }
    }
}